vowpalwabbit/gd.cc

```diff
@@ -253,7 +253,7 @@ inline void audit_feature(audit_results& dat, const float ft_weight, const uint6
     tempstream << ':' << (index >> stride_shift) << ':' << ft_weight << ':'
                << trunc_weight(weights[index], (float)dat.all.sd->gravity) * (float)dat.all.sd->contraction;
 
-    if (dat.all.adaptive)
+    if (stride_shift > 0) // adaptive
```

Comment:
We can rely on stride shift indicating there is adaptive?

Reply:
Not sure yet, this work is still in progress.

Reply:
Ah okay no problem